draft-ietf-quic-multipath.md

```diff
@@ -998,7 +998,7 @@ All the new frames MUST only be sent in 1-RTT packet, and MUST NOT
 use other encryption levels.
 
 If an endpoint receives multipath-specific frames from packets of
-other encryption levels, it MUST return MP_PROTOCOL_VIOLATION
+other encryption levels, it MUST return FRAME_ENCODING_ERROR
```

Comment:
Maybe we should change "packets of other encryption levels" to "types of packets."

In RFC 9000, we talk about "type of packets" when clarifying when the frames can be used; see [section 12.4](https://www.rfc-editor.org/rfc/rfc9000#name-frames-and-frame-types).

With the understanding that this is not the problem that the PR is trying to address, it might make sense to adjust the phrasing.

This applies to the paragraph right above as well. It would be suffice to drop the MUST NOT there, it would suffice to just state "All the new frames MUST only be sent in 1-RTT."